src/keys.ts

```diff
@@ -15,8 +15,9 @@ export namespace JWTAuthenticationBindings {
   );
 }
 
-export namespace OtherServicesBindings {
-  export const HASH_PASSWORD = BindingKey.create<HashPassword>(
-    'services.hash_password',
+export namespace PasswordHasherBindings {
+  export const PASSWORD_HASHER = BindingKey.create<PasswordHasher>(
+    'services.password.hasher',
```

Comment:
nitpick: Is it possible to rename `services.password.hasher` to `services.hasher.password` so that it's in the namespace `hasher` like `services.hasher.round` below?

Reply:
The difference is `services.password.hasher` maps to the hasher class, while `services.hasher.rounds` maps to a class member, maybe I should rename `services.password.hasher` to be `services.hasher`, does it make more sense?

Reply:
Your proposal sounds great, but feel free to ignore.